test/test_habitat_env.py

```diff
@@ -211,6 +211,7 @@ def test_rl_vectorized_envs():
         assert len(rewards) == num_envs
         assert len(dones) == num_envs
         assert len(infos) == num_envs
+        assert envs.render(mode="rgb_array")
```

Comment:
Should this be `"rgb"`, instead of `"rgb_array"`. 

Also on a note of testing this, I don't think adding an assert check here is a good idea because the render method will display a window with tiled images. Just testing it locally and posting a screenshot would be useful for testing this functionality.

Reply:
Thanks for commenting on this!
1) It has to be `"rgb_array"` because render function looks like:
https://github.com/facebookresearch/habitat-api/blob/37fc4de0216fd3287cb08f9f7b6e7349c1f286b6/habitat/core/vector_env.py#L406-L424
So it does not know anything about `"rgb"` mode. The thing is, that you have to pass `"rgb"` mode only to worker, not to VectorEnv's render function. If you want consistency at this point, then we have to change render function by replacing all mentions of `"rgb_array"` by `"rgb"`.
2) Well, no it won't display a window. It does that when mode is set to 'human'. In the case of `"rgb_array"` mode it will just render this image and return it as an array, without opening a window. Anyway, I can save image returned by this function and post it here if it is needed.

Reply:
Thanks, sorry I had forgotten the exact working of the render method, can you add a message along with the assert check, `assert envs.render(mode="rgb_array"), "vector env render is broken"` and just add an image to the PR, it will be good to go then.

Reply:
Sure